include/instrumentation/syscalls/headers/syscalls_integers_override.h

```diff
@@ -38,9 +38,13 @@ SC_LTTNG_TRACEPOINT_ENUM(lttng_fcntl_cmd_flags,
 		ctf_enum_value("F_GETOWN", F_GETOWN)
 		ctf_enum_value("F_SETSIG", F_SETSIG)
 		ctf_enum_value("F_GETSIG", F_GETSIG)
+
+#ifdef CONFIG_COMPAT
```

Comment:
Looking at uapi/asm-generic/fcntl.h, it shows:

#ifndef CONFIG_64BIT
#ifndef F_GETLK64
#define F_GETLK64       12      /*  using 'struct flock64' */
#define F_SETLK64       13
#define F_SETLKW64      14
#endif
#endif

Shouldn't we ifndef CONFIG_64BIT rather than ifdef on CONFIG_COMPAT ?